src/components/FilterTags/FilterTags.js

```diff
@@ -121,7 +197,7 @@ const FilterTags = () => {
                   {filters[filterBy].map(filter => (
                     <li key={filter} className="p-filter-panel__item">
                       <button
-                        onClick={e => handleFilterClick(e)}
+                        onClick={e => addActiveFilter(e, filter, filterBy)}
```

Comment:
I'd expect to be able to click the filter again to remove it from the set so the original name might be better.